### PR TITLE
[Windows] Add WIX binaries folder to the PATH

### PR DESCRIPTION
**Pull request by erik-bershel**
# Description

Adding `WIX` binaries folder to the `PATH` environment variable. 

#### Related issue:

https://github.com/actions/runner-images/issues/9551

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/windows/scripts/build/Install-Wix.ps1 [hunk 1]
```diff
@@ -5,4 +5,16 @@
...
 
+# Add WIX binaries to PATH
+$wixBinPath = Join-Path -Path $env:WIX -ChildPath 'bin'
+if (Test-Path $wixBinPath) {
+    $currentPath = [System.Environment]::GetEnvironmentVariable("PATH", "Machine")
+    $newPath = $currentPath + ";$wixBinPath"
+    Write-Host "Add $wixBinPath to PATH"
+    [Environment]::SetEnvironmentVariable("PATH", $newPath, "Machine")
+    $env:Path += ";$wixBinPath"
+}else{
```
Comment (mikhailkoliada) [inline images/windows/scripts/build/Install-Wix.ps1:16]:
```suggestion
} else {
```

File: images/windows/scripts/build/Install-Wix.ps1 [hunk 2]
```diff
@@ -5,4 +5,16 @@
 
 Install-ChocoPackage wixtoolset -ArgumentList "--force"
 
+# Add WIX binaries to PATH
+$wixBinPath = Join-Path -Path $env:WIX -ChildPath 'bin'
+if (Test-Path $wixBinPath) {
+    $currentPath = [System.Environment]::GetEnvironmentVariable("PATH", "Machine")
+    $newPath = $currentPath + ";$wixBinPath"
+    Write-Host "Add $wixBinPath to PATH"
```
Comment (mikhailkoliada) [inline images/windows/scripts/build/Install-Wix.ps1:13]:
```suggestion
```

File: images/windows/scripts/build/Install-Wix.ps1 [hunk 3]
```diff
@@ -5,4 +5,16 @@
 
 Install-ChocoPackage wixtoolset -ArgumentList "--force"
```
Comment (mikhailkoliada) [inline images/windows/scripts/build/Install-Wix.ps1:6]:
```suggestion
Install-ChocoPackage wixtoolset -ArgumentList "--force"

$wixBinPath = Join-Path -Path $env:WIX -ChildPath 'bin'
```

File: images/windows/scripts/build/Install-Wix.ps1 [hunk 4]
```diff
@@ -5,4 +5,16 @@
...
 
+# Add WIX binaries to PATH
```
Comment (mikhailkoliada) [inline images/windows/scripts/build/Install-Wix.ps1:8]:
```suggestion
Update-Environment

```

File: images/windows/scripts/build/Install-Wix.ps1 [hunk 5]
```diff
@@ -5,4 +5,16 @@
...
+$wixBinPath = Join-Path -Path $env:WIX -ChildPath 'bin'
```
Comment (mikhailkoliada) [inline images/windows/scripts/build/Install-Wix.ps1:9]:
```suggestion

$wixBinPath = Join-Path -Path $env:WIX -ChildPath 'bin'
```

File: images/windows/scripts/tests/Wix.Tests.ps1 [hunk 6]
```diff
@@ -8,4 +8,8 @@ Describe "Wix" {
     It "Wix Toolset version from registry" {
       $version | Should -Not -BeNullOrEmpty
     }
+
+    It "Wix binaries are available via env:Path" {
```
Comment (mikhailkoliada) [inline images/windows/scripts/tests/Wix.Tests.ps1:12]:
```suggestion
    It "WIX binaries are available" {
```

File: images/windows/scripts/build/Install-Wix.ps1 [hunk 7]
```diff
@@ -5,4 +5,16 @@
 
 Install-ChocoPackage wixtoolset -ArgumentList "--force"
 
+Update-Environment
+
+$wixBinPath = Join-Path -Path $env:WIX -ChildPath 'bin'
```
Comment (mikhailkoliada) [inline images/windows/scripts/build/Install-Wix.ps1:10]:
```suggestion
$wixBinPath = Join-Path -Path $env:WIX -ChildPath "bin"
```

**Comment by lzandman** [comment]
Ah, here's the answer to [my question](https://github.com/actions/runner-images/pull/9567#issuecomment-2025287016). You hijacked my pull request #9567. No credits for me ;-) Ah, well, as long as the issue gets fixed.

**Comment by erik-bershel** [comment]
Hi @lzandman!

> I don't understand. This PR fixes a real issue a user is experiencing. Is the plan to not fix this issue? Or is it just that my code isn't OK?

As I mentioned, I do not have the right to force changes to third-party PRs, and you did not respond to the comment for about a day, it was in connection with this that I opened an additional PR from my branch, which was created separately from your request. 

> Ah, here's the answer to https://github.com/actions/runner-images/pull/9567#issuecomment-2025287016. You hijacked my pull request https://github.com/actions/runner-images/pull/9567. No credits for me ;-) Ah, well, as long as the issue gets fixed.

If this is a fundamental point for you, then I can provide logs of my repositories and browser, after analysing which it will be possible to draw conclusions about the uniqueness of individual solutions. Separately, I should note that I understand that the situation may look quite ambiguous, and that you had the right to draw the conclusions that you did. But please understand us - not all work, even in a public repository, is visible to everyone, and not all events are transparent to each individual participant. I'm sorry if my actions have upset you because, again, I truly value the contributions of everyone in the community. I'll be glad to add you to this PR as a co-author if that's okay with you.

**Comment by lzandman** [comment]
> As I mentioned, I do not have the right to force changes to third-party PRs
This was the part that I initially didn't understand you meant. I now get why you created that separate PR. And indeed, it took some time for me to react. I have a life outside of Github too, LOL ;-)

> If this is a fundamental point for you

No, this isn't a problem. Also, I'm not upset. I added a smiley ;-) in my comment above, that maybe you did not see. All is fine. Keep up the good work!

**Review by mikhailkoliada — APPROVED** [review]

**Pull request merged**